pkg/action/install.go

```diff
@@ -120,7 +124,8 @@ type ChartPathOptions struct {
 // NewInstall creates a new Install object with the given configuration.
 func NewInstall(cfg *Configuration) *Install {
 	return &Install{
-		cfg: cfg,
+		cfg:            cfg,
+		RegistryClient: cfg.RegistryClient,
```

Comment:
Is this needed? Is RegistryClient not present in cfg itself? I understand cfg is not public, but it the registry client can be extracted out of it and passed down to underlying methods.

Reply:
The `cfg` is not available outside `package action`, specifically, it's not available in `package main` where the `downloader.Manager` is assembled during the `runInstall` function:
https://github.com/helm/helm/blob/8888252c1837b3e40ce4d7dad4f94cbc1385c3f8/cmd/helm/install.go#L226